dashboards-observability/query_manager/ast/builder/query_builder.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export interface QueryBuilder<T> {
+  build: () => T;
+}
```

Comment:
We can move this file to constant.

Reply:
For this and later similar comments, I'm thinking to keep all these in query manager only as they are all specific file/types for query manager, which will be sharing not only by observability plugin.